js/index.js

```diff
@@ -0,0 +1,240 @@
+};
+
+const keyboardContainerCreation = () => {
+  const keyboardContainer = document.createElement('div');
+  keyboardContainer.classList.add('keyboard-container');
+
+  switch (localStorage.language) {
+    case 'en':
+      arrayOfLatinCharacters.forEach((obj) => {
+        keyboardContainer.appendChild(keyContainersCreation(obj));
+      });
+      break;
+
+    case 'ru':
+      arrayOfCyrillicCharacters.forEach((obj) => {
+        keyboardContainer.appendChild(keyContainersCreation(obj));
+      });
+      break;
+
+    default:
+  }
+
+  document.querySelector('body').appendChild(keyboardContainer);
+};
+
+const deleteKeyboardContainer = () => {
+  if (document.querySelector('.keyboard-container')) {
+    document.querySelector('.keyboard-container').remove();
+  }
+};
+
+if (!localStorage.language) {
+  localStorage.setItem('language', 'en');
+}
+
+textAreaCreation();
+keyboardContainerCreation();
+
+document.addEventListener('keydown', (e) => {
+  const arrayOfKeyContainers = document.querySelectorAll('.key-container');
+  const textArea = document.querySelector('.text-area');
+
+  e.preventDefault();
+
+  if (e.altKey && e.shiftKey) {
+    isShiftAndAltPressed = true;
+  }
+
+  arrayOfKeyContainers.forEach((keyContainer) => {
+    if (keyContainer.attributes.code.value === e.code) {
+      keyContainer.classList.add('pressed-key-container');
+
+      if (keyContainer.attributes.functionKey.value !== 'true') {
+        if (isCapsLockPressed && e.shiftKey === false) {
+          textArea.value += keyContainer.attributes.key.value.toUpperCase();
+        } else if (isCapsLockPressed && e.shiftKey && keyContainer.attributes.altKey.value !== 'undefined') {
+          textArea.value += keyContainer.attributes.altKey.value;
+        } else if (isCapsLockPressed && e.shiftKey) {
+          textArea.value += keyContainer.attributes.key.value.toLowerCase();
+        } else if (!e.shiftKey) {
+          textArea.value += keyContainer.attributes.key.value.toLowerCase();
+        } else if (e.shiftKey && keyContainer.attributes.altKey.value !== 'undefined') {
+          textArea.value += keyContainer.attributes.altKey.value;
+        } else if (e.shiftKey) {
+          textArea.value += keyContainer.attributes.key.value;
+        }
+      } else {
+        switch (keyContainer.attributes.key.value) {
+          case 'CapsLock':
+            isCapsLockPressed = !isCapsLockPressed;
+
+            document.querySelector('.caps-lock-indicator').classList.toggle('caps-lock-indicator_active');
+            break;
+          case 'Backspace':
+            textArea.value = textArea.value.slice(0, textArea.value.length - 1);
+            break;
+          case 'Space':
+            textArea.value += ' ';
+            break;
+          case 'Tab':
+            textArea.value += '        ';
+            break;
+          case 'Enter':
+            textArea.value += '\n';
+            break;
+
+          default:
+        }
+      }
+    }
```

Comment:
logic is similar, can you move it into another function and reuse in different places?